src/timepicker/timepicker.component.ts

```diff
@@ -99,18 +107,14 @@ export class TimepickerComponent
   @Input() max: Date;
 
   /** emits true if value is a valid date */
-  @Output() isValid: EventEmitter<boolean> = new EventEmitter();
+  @Output() isValid = new EventEmitter<boolean>();
 
   // ui variables
   hours: string;
   minutes: string;
   seconds: string;
   meridian: string;
 
-  get isSpinnersVisible(): boolean {
```

Comment:
removing public properties of components considered a breaking change and require major version release
and it will not happen cuz of spinner visibility 

instead mark it as `@deprecated` and return `this.isActive`

plus isActive - not the best name in this case, usually it's used to say that component has focus